GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -888,8 +892,9 @@ ProcessingResult QtQuickAdapter::processWalletData(bs::message::SeqId msgId
    }
    QVector<QVector<QString>> addresses;
    for (const auto& addr : response.used_addresses()) {
-      addresses.append({ QString::fromStdString(addr.address())
-      , QString::fromStdString(addr.comment().empty() ? addr.index() : addr.comment()) });
+      addresses.append({ QString::fromStdString(addr.address()),
+                         QString::fromStdString(addr.comment()),
```

Comment:
Keep using the existing style (commas before the next arg are carried over to the beginning of the new line).